plugins/org/src/components/Cards/Group/MembersList/MembersListCard.stories.tsx

```diff
@@ -109,44 +122,39 @@ const catalogApi = (items: Entity[]) => ({
   getEntities: () => Promise.resolve({ items }),
 });
 
+// Removed the MemoryRouter and added the WrapinTestApp decorator for proper routing context
```

Comment:
There's no need to leave a comment with the code changes. It can be done for context on the PR in GitHub, but the code should generally reflect the current state

Reply:
Removed this comment line as well